### DISCUSSION TITLE
WAN -> LAN

### DISCUSSION THREAD

**Discussion by xujiaqing2016** [rustdesk/rustdesk#1728] [Q&A]
自建了服务器，在局域网里面可以正常使用，但是外网访问就提示无法连接到中继服务器，有公网IP，设置的DMZ，端口已经打开，也设置过IP转发，但是效果都一样。不知道从哪里开始排查？

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server/issues/24#issuecomment-1234029480

**Reply by fentouxungui** [reply]
客户端试试：  ID服务器和中继服务器都要填写

**Comment by snowdream** [comment]
内外网客户端全部统一使用公网IP或域名时，服务器才会自动转换中继数据包。

https://www.cnblogs.com/System-admin/p/17659074.html
